run_tests.py

```diff
@@ -0,0 +1,5 @@
+from modules.tests import test_MsSqlDataSource
+import unittest
+
+suite = unittest.TestLoader().loadTestsFromModule(test_MsSqlDataSource)
+unittest.TextTestRunner(verbosity=2).run(suite)
```

Comment:
What level of verbosity does `2` represent?

Reply:
added the following comment: https://stackoverflow.com/a/1322648/8030743